superset/utils/webdriver.py

```diff
@@ -42,6 +43,12 @@
     from flask_appbuilder.security.sqla.models import User
 
 
+class DashboardStandaloneMode(int, Enum):
+    HIDE_NAV = (1,)
+    HIDE_NAV_AND_TITLE = (2,)
+    REPORT = (3,)
```

Comment:
```suggestion
    HIDE_NAV = 1
    HIDE_NAV_AND_TITLE = 2
    REPORT = 3
```

Otherwise the type is not `int`, but `Tuple[int]`.